apps/api/src/app/auth/services/auth.service.ts

```diff
@@ -1,12 +1,7 @@
 import { Inject, Injectable } from '@nestjs/common';
 import { MemberEntity, SubscriberEntity, UserEntity } from '@novu/dal';
-import {
-  AuthProviderEnum,
-  AuthenticateContext,
-  ISubscriberJwt,
-  UserSessionData,
-} from '@novu/shared';
-import { IAuthService } from './auth.service.interface';
+import { AuthProviderEnum, AuthenticateContext, ISubscriberJwt, UserSessionData } from '@novu/shared';
+import { IAuthService } from '@novu/application-generic';
```

Comment:
This one interface needs to stay in app-generic because its reused in EE auth.

We need to have common auth interface for private and public authentication so it makes sense.